packages/gasket-fetch/lib/index.d.ts

```diff
@@ -0,0 +1,16 @@
+async function fetch(input: RequestInfo, init?: RequestInit): Promise<Response>;
+
+export default fetch
+export { fetch }
+
+type GasketHeaders = Headers;
+export { GasketHeaders as Headers }
+
+type GasketRequest = Request;
+export { GasketRequest as Request }
+
+type GasketResponse = Response;
+export { GasketResponse as Response }
+
+type GasketAbortController = AbortController;
+export { GasketAbortController as AbortController }
```

Comment:
Is this working around some limitation, or could you just do:

```suggestion
export { Headers, Request, Response, AbortController }
```

Reply:
Yea, dunno. I must've had something weird with my setup, because your example wasn't working for me before. Looks good now though.